lib/openstax/exchange/client/fake_client.rb

```diff
@@ -0,0 +1,77 @@
+require 'securerandom'
+require 'uri'
+
+module OpenStax
+  module Exchange
+    module Client
+      class FakeClient
+        include ClientInstance
```

Comment:
Both `RealClient` and `FakeClient` implement an interface, which I'm calling `ClientInstance`.  So I can either include `ClientInstance` or extend it, and I thought this would be the more idiomatic way to handle it.

I guess what you're really saying is that I should include `#create_identifier` and `#create_multiple_choice` in `ClientInstance`, since that would make sense.  :)

The reason for having the mixin (or base class if you like that approach) is to document the implied interface.  This allows, among other things, shared example specs like "acts like client instance" to ensure classes implement the required methods.

So I think it is A Good Thing.